eip712/messages.py

```diff
@@ -156,6 +156,13 @@ def _struct_hash_(self) -> HexBytes:
         message = _prepare_data_for_hashing(self._body_["message"])
         return HexBytes(hash_eip712_message(types, message))
 
+    @property
+    def _encoded_struct_(self) -> HexBytes:
```

Comment:
Is `_struct_hash_` equal to `keccak(_encoded_struct_)`?

Reply:
yes, it seems like it